packages/editor/src/components/collab-sidebar/add-comment.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useState, useEffect } from '@wordpress/element';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	Button,
+	TextControl,
+} from '@wordpress/components';
+import { store as blockEditorStore } from '@wordpress/block-editor';
+import { store as coreStore } from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import { sanitizeCommentString } from './utils';
+
+/**
+ * Renders the new comment form.
+ *
+ * @param {Object}   root0          The component props.
+ * @param {Function} root0.onSubmit Function to add new comment.
+ */
+export function AddComment( { onSubmit } ) {
+	// State to manage the comment thread.
+	const [ inputComment, setInputComment ] = useState( '' );
+
+	const currentUserData = useSelect( ( select ) => {
+		return select( coreStore ).getCurrentUser();
+	}, [] );
+
+	const useDefaultAvatar = () => {
+		const { avatarURL: defaultAvatarUrl } = useSelect( ( select ) => {
+			const { getSettings } = select( blockEditorStore );
+			const { __experimentalDiscussionSettings } = getSettings();
+			return __experimentalDiscussionSettings;
+		} );
+		return defaultAvatarUrl;
+	};
+
+	const defaultAvatar = useDefaultAvatar();
```

Comment:
This can likely be moved out of the `useDefaultAvatar` hook:

```suggestion
	const { avatarURL: defaultAvatar } = useSelect( ( select ) => {
		const { getSettings } = select( blockEditorStore );
		const { __experimentalDiscussionSettings } = getSettings();
		return __experimentalDiscussionSettings;
	} );
```